templates/records/record_disambiguation_page.html

```diff
@@ -24,8 +24,8 @@ <h2 class="sr-only">Associated records</h2>
             <ul class="card-group--list-style-none" data-container-name="associated-records"
                 id="analytics-associated-records">
 
-                {% for page in pages %}
-                    {% include 'includes/card-group-record-summary-no-image.html' with record=page %}
+                {% for record_page in record_pages %}
+                    {% include 'includes/card-group-record-summary-no-image.html' with record=record_page %}
                 {% endfor %}
```

Comment:
With my proposed name (above), this would become:

```suggestion
                {% for record in paginator_page %}
                    {% include 'includes/card-group-record-summary-no-image.html' %}
                {% endfor %}
```

Reply:
Done to reflect `record_results_page`